src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/analysis/AnalysisDetailsPageIT.java

```diff
@@ -297,7 +297,7 @@ public void testTabRoutingAnalysisCompleted() throws URISyntaxException, IOExcep
 		page = AnalysisDetailsPage.initPage(driver(), 4L, "settings");
 		assertTrue("Page title should equal", page.compareTabTitle("Details"));
 
-		page = AnalysisDetailsPage.initPage(driver(), 4L, "settings/details");
+		page = AnalysisDetailsPage.initPage(driver(), 4L, "settings");
```

Comment:
I would prefer this to be reverted back to `settings/details`, or we just remove this and the following line as it is redundant.

Reply:
Updated in 9da8e9f